crates/bevy_ecs/src/schedule/executor_parallel.rs

```diff
@@ -145,29 +101,25 @@ impl ParallelSystemExecutor for ParallelExecutor {
                 self.system_metadata[*dependency].dependants.push(dependant);
             }
         }
+
+        self.update_archetypes(systems, world);
```

Comment:
This isn't needed here. What it does was done above in an explicit way in order to reduce the number of iterations the executor does when systems change (which only happens at startup right now, but, future-proofing).

Reply:
Good call!